server/routes/api.js

```diff
@@ -909,7 +922,7 @@ function inParam (sql, arr) {
 // route middleware to make sure a user is logged in
 function isLoggedIn(req, res, next) {
   if (req.method == 'GET') { 
-    if (apiSecurity || req.url.match(/capcodes/i)) { //check if Secure mode is on, or if the route is a capcode route
+    if (apiSecurity || req.url.match(/capcodes/i) && !(req.url.match(/agency/))) { //checkc if Secure mode is on, or if the route is a capcode route
```

Comment:
```suggestion
    if (apiSecurity || req.url.match(/capcodes/i) && !(req.url.match(/agency$/))) { //check if Secure mode is on, or if the route is a capcode route
```

Typo there, and also modifying the check to match absolutely on /agency, otherwise URLs like `/agency/RFS` will return capcodes for that agency